vms/platformvm/txs/fee/dynamic_config.go

```diff
@@ -0,0 +1,57 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package fee
+
+import (
+	"fmt"
+
+	"github.com/ava-labs/avalanchego/snow"
+	"github.com/ava-labs/avalanchego/utils/constants"
+	"github.com/ava-labs/avalanchego/utils/units"
+
+	commonfees "github.com/ava-labs/avalanchego/vms/components/fees"
+)
+
+var (
+	eUpgradeDynamicFeesConfig = commonfees.DynamicFeesConfig{
+		FeeRate: commonfees.Dimensions{
+			1 * units.NanoAvax,
+			2 * units.NanoAvax,
+			3 * units.NanoAvax,
+			4 * units.NanoAvax,
+		},
+
+		BlockMaxComplexity: commonfees.Max,
```

Comment:
these values don't really matter.  We'll change these parameters when introducing the dynamic multifee and then we'll tuno those parameters (which will actually see production)